pyhf/__init__.py

```diff
@@ -1,8 +1,11 @@
 import logging
+logging.basicConfig()
```

Comment:
as a library, we should probably not be configuring logging. if we add a CLI tool at some point we might do it there